entries/i/instant-gaming.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Instant-Gaming": {
+    "domain": "instant-gaming.com",
+    "tfa": [
+      "totp"

```

Comment:
Please attach some screenshots of the 2FA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

Reply:
Of course, can you give me a commit or other to have an example. Because I didn't find an example ^^

Reply:
[Here is a PR](https://github.com/2factorauth/twofactorauth/pull/6401) where I have included a screenshot of the login screen asking for a code, the screenshot shows SMS verification which matches what I put in the JSON.

Secondly, [here is a recent one](https://github.com/2factorauth/twofactorauth/pull/8034) where I have included both the setting in the account settings and what you see when you login. The bottom-right screenshot confirms its a code emailed to you.

I have blurred any sensitive information before attaching these images.

Reply:
@fletcherdodd i have add image how to setup 2fa, i hope it's ok :)

Reply:
That should be more than adequate! Nice work.